homeassistant/components/enphase_envoy/const.py

```diff
@@ -33,7 +33,7 @@
         key="seven_days_production",
         name="Last Seven Days Energy Production",
         native_unit_of_measurement=UnitOfEnergy.WATT_HOUR,
-        state_class=SensorStateClass.TOTAL_INCREASING,
+        state_class=SensorStateClass.TOTAL,
```

Comment:
elupus is correct, if this is a sliding window, it shouldn't have a state class at all here.

I suggest to remove it

```suggestion
```